app/models/broadcast.rb

```diff
@@ -23,13 +24,15 @@ def get_inner_body(content)
   private
 
   def single_active_announcement_broadcast
+    # TODO: [@thepracticaldev/delightful]: Move this logic into the Announcement model
```

Comment:
👍 nice! can't wait to get this all out of here and into the appropriate model :)